src/aac/plugins/gen_gherkin_behaviors/__init__.py

```diff
@@ -31,3 +33,15 @@ def get_commands() -> list[AacCommand]:
     ]
 
     return plugin_commands
+
+
+@hookimpl
+def get_base_model_extensions() -> str:
+    """
+    Returns the CommandBehaviorType modeling language extension to the plugin infrastructure.
+
+    Returns:
+        string representing yaml extensions and data definitions employed by the plugin
+    """
+    with resources.open_text(__package__, "gen_gherkin_behaviors.yaml") as plugin_model_file:
```

Comment:
Since this is a common pattern, we might do better extracting this into a helper function in `plugins/__init__.py` and just call it passing the name of the package and the name of the file - I think in that way it will be cleaner and less error prone.

Reply:
I'd be more inclined to agree if this wasn't a two-line boilerplate pattern. 

There is also the consideration of injecting a circular dependency if a plugin relies on the `plugins` package, but the plugin is initialized as part of the `plugins` initialization where the plugins are registered.

Reply:
A two-line boilerplate pattern is still more than a one-line boilerplate pattern - i.e. a function call.

Regarding putting it into `plugins/__init__.py` causing circular import issues, that makes sense but couldn't that be resolved by putting it into another file in the `plugins/` directory? Maybe we can put it in `plugins/common.py`, or something similar to avoid circular imports?

Reply:
Ok, I've implemented a new module and function to thin the boilerplate a little.